Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -233,5 +239,27 @@ public async Task<IActionResult> GetNewsfeedAsync()
             return NoContent();
         }
 
+        [HttpPost("UploadPicture"), DisableRequestSizeLimit]
+        public async Task<ActionResult> UploadPicture() 
+        {
+            try {
+                if (Request.Form.Files[0] is IFormFile file)
```

Comment:
just use model binding

Reply:
UploadPicture now takes a file and userId parameter